ConfigSpace/hyperparameters.py

```diff
@@ -991,37 +991,43 @@ def get_num_neighbors(self, value: Union[int, float, str]) -> int:
         """
         returns the number of existing neighbors in the sequence
         """
-        if value == self.sequence[0] or value == self.sequence[-1]:
+        if value == self.sequence[0] and value == self.sequence[-1]:# check if there is only one value
+            return 0
+        elif value == self.sequence[0] or value == self.sequence[-1]:
```

Comment:
I think there is no check that the values in the sequence are unique. Could you please add that?